docs/sql-performance-tuning.md

```diff
@@ -229,7 +229,9 @@ The "REPARTITION_BY_RANGE" hint must have column names and a partition number is
     SELECT /*+ REPARTITION_BY_RANGE(c) */ * FROM t
     SELECT /*+ REPARTITION_BY_RANGE(3, c) */ * FROM t
     SELECT /*+ REBALANCE */ * FROM t
+    SELECT /*+ REBALANCE(3) */ * FROM t
```

Comment:
Seems like the `REBALANCE` hint is not documented above. Can we add it?

Reply:
yea, add some words in `docs/sql-performance-tuning.md` and `docs/sql-ref-syntax-qry-select-hints.md`.